static/js/app.js

```diff
@@ -1,7 +1,7 @@
 // HMIS dashboard
 
 var t0 = performance.now();
-var url = 'http://localhost:5000/api'
+var url = 'https://hmis-dashboard.herokuapp.com/api'
```

Comment:
We should either change this to a relative url (i.e. just `'api'`) or set this with an environment variable if we need the full url

Reply:
I'll merge for now and create an issue for this after.